app/params/params.go

```diff
@@ -4,4 +4,10 @@ package params
 const (
 	StakePerAccount           = "stake_per_account"
 	InitiallyBondedValidators = "initially_bonded_validators"
+
+	DefaultWeightMsgCreateDenom      int = 100
```

Comment:
Don't those belong in x/tokenfactory and not app?

I mean, if anyone else wants to add these sims in a project they import, they probably don't want to import the app/* dirs.

Reply:
Yeah, they can just import these sims in there project (add sims module manager in app.sm) and it can run normally (no need to import app/*)